src/config/config.js

```diff
@@ -0,0 +1,22 @@
+import Sequelize from 'sequelize';
+import dotenv from "dotenv";
+
+dotenv.config();
+const db_con = process.env.DB_URL;
+export const sequelize = new Sequelize(
+    db_con,
+     {
+        host: 'localhost',
+        dialect: 'postgres',
+   pool:{
+         max:5,
+         min:0,
+         idle:10000
+   },
+   logging: false
+ });
+sequelize.authenticate().then(() => {
+     console.log('Connection has been established successfully.');
+}).catch(err => {
+     console.error('Unable to connect to the database:', err);
+});
```

Comment:
Add a new line to look good

Reply:
i don't understand what you are trying to say